cypress/e2e/gui/customer.spec.cy.js

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+
+/*
+  Retorna à lista de clientes ao clicar no botão "Voltar" *
+  Exibe o rodapé com o texto e links corretos *
+  Exibe o cabeçalho com um título, alternador de tema e um campo de entrada de texto *
+  Abre e fecha o messenger *
+  Garante que todos os campos do messenger são obrigatórios e que o primeiro está focado *
+  Mostra e oculta uma mensagem de sucesso ao enviar o formulário do messenger com sucesso *
+  Limpa todos os campos do formulário do messenger ao preenchê-los, fechar o messenger e abri-lo novamente *
+  Mostra as colunas Nome da Empresa e Ação, e oculta as colunas ID, Indústria, Número de Funcionários e Tamanho em um viewport móvel *
+*/
+
+describe('EngageSphere Testing App', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('should display the banner on the first visit', () => {
+    cy.get('body')
+      .get('.CookieConsent_banner__UHArL')
+      .should('be.visible')
+
+    cy.contains('Accept').should('be.visible')
```

Comment:
Segue uma opção que torna a seleção mais específica.

```suggestion
    cy.contains('button', 'Accept').should('be.visible')
```

Dessa forma, garantimos que a verificação é sobre o elemento certo, caso haja outra _tag_ HTML com o mesmo texto.

Reply:
O mesmo vale para outros locais onde o `cy.contains` esteja sendo utilizado somente com um argumento.

A propósito, recomendo a leitura do seguinte conteúdo no blog TAT: [Como identificar um elemento por seu texto com Cypress](https://talkingabouttesting.com/2021/02/13/como-identificar-um-elemento-por-seu-texto-com-cypress/).